utils/gha-runners/createNamespaceConfig.sh

```diff
@@ -0,0 +1,152 @@
+}
+
+function create_fsdax() {
+	local size=$1
+	local size_option="-s $size"
+
+	if [ -z "$size" ]; then
+		size_option=""
+	fi
+
+	local cmd="sudo ndctl create-namespace --mode fsdax ${size_option} -r ${FS_DAX_R} -f"
+	result=$(${cmd})
+	if [ $? -ne 0 ]; then
+		exit 1;
+	fi
+	jq -r '.blockdev' <<< $result
+}
+
+#Namespaces size must align with interleave-width which is derived from number of dimms connected to a socked.
+function check_alignment() {
+	local size=$1
+	local interleave_width=$(sudo ipmctl show -dimm -socket 1 | grep "0x1" | wc -l)
+	local size_alignment=$(expr $size % $interleave_width)
+
+	if [ "$size_alignment" -gt "0" ]; then
+		size=$(expr $size - $size_alignment + $interleave_width)
+	fi
+
+	echo "${size}G"
+}
+
+while getopts ":h-:" optchar; do
+	case "${optchar}" in
+		-)
+		case "$OPTARG" in
+			help) usage $0 && exit 0 ;;
+			conf-path=*) CONF_PATH="${OPTARG#*=}" ;;
+			*) echo "Invalid argument '$OPTARG'"; usage $0 && exit 1 ;;
+		esac
+		;;
+		h) usage $0 && exit 0 ;;
+		*) echo "Invalid argument '$OPTARG'"; usage $0 && exit 1 ;;
+	esac
+done
+
+# Remove existing namespaces.
+clear_namespaces
+
+#fsdaxes are created on separete region than devdaxes and it has 256GB of storage.
+FS_DAX_SIZE=$(check_alignment 100)
+
+BIG_NS_SIZE=$(check_alignment 55)
+SMALL_NS_SIZE=$(check_alignment 4)
+
+#Creating devdax namespaces.
+trap 'echo "ERROR: Failed to create namespaces"; clear_namespaces; exit 1' ERR SIGTERM SIGABRT
+
+		dax[0]=$(create_devdax 4k $SMALL_NS_SIZE)
+		dax[1]=$(create_devdax 4k $SMALL_NS_SIZE)
+		dax[2]=$(create_devdax 2m $SMALL_NS_SIZE)
+		dax[3]=$(create_devdax 2m $SMALL_NS_SIZE)
+		dax[4]=$(create_devdax 4k $BIG_NS_SIZE)
+		dax[5]=$(create_devdax 4k $BIG_NS_SIZE)
+		dax[6]=$(create_devdax 2m $BIG_NS_SIZE)
+		dax[7]=$(create_devdax 2m $BIG_NS_SIZE)
+		dax[8]=$(create_devdax 2m $SMALL_NS_SIZE)
+
+#Creating mountpoint.
+trap 'echo "ERROR: Failed to create mountpoint"; clear_namespaces; exit 1' ERR SIGTERM SIGABRT
+pmem_name=$(create_fsdax $FS_DAX_SIZE)
+
+if [ ! -d "${MOUNT_POINT}" ]; then
+	sudo mkdir ${MOUNT_POINT}
+fi
+
+if ! grep -qs "${MOUNT_POINT} " /proc/mounts; then
+	sudo mkfs.ext4 -F /dev/${pmem_name}
+	sudo mount -o dax /dev/${pmem_name} ${MOUNT_POINT}
+fi
+
+#Changing file permissions.
+sudo chmod 777 ${MOUNT_POINT} || true
+sudo chmod 777 /dev/dax* || true
+sudo chmod a+rw /sys/bus/nd/devices/region*/deep_flush
+sudo chmod +r /sys/bus/nd/devices/ndbus*/region*/resource
+sudo chmod +r  /sys/bus/nd/devices/ndbus*/region*/dax*/resource
+
+#Print created namespaces.
+ndctl list -X | jq -r '.[] | select(.mode=="devdax") | [.daxregion.devices[].chardev, "align: "+(.align/1024|tostring+"k"), "size: "+(.size/1024/1024/1024|tostring+"G") ]'
+ndctl list | jq -r '.[] | select(.mode=="fsdax") | [.blockdev, "size: "+(.size/1024/1024/1024|tostring+"G") ]'
+
+#If config dir does not exists, create it.
+mkdir --parents ${CONF_PATH}
+
+#Create config file for unittests.
+cat >${CONF_PATH}/testconfig.sh <<EOL
```

Comment:
We also need py tests.
Please add valid testconfig.py file - see testconfig.py.example.